client/src/components/VideoCard.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { AspectRatio, Flex, Box, Heading, IconButton } from "@chakra-ui/react";
+import './VideoCard.css';
+import { FaThumbsUp, FaThumbsDown, FaBan } from 'react-icons/fa';
+
+const VideoCard = ({ videoData, videosArr, setVideosArr }) => {
+	const [ratingUp, setRatingUp] = useState(false);
+	const [ratingDown, setRatingDown] = useState(false);
+	const [ratingNr, setRatingNr] = useState(videoData.rating);
+
+	const incrementRating = () => {
+		if (!ratingUp) {
+			setRatingUp(!ratingUp);
+			setRatingNr(ratingNr + 1);
+		} else {
+			setRatingUp(!ratingUp);
+			setRatingNr(ratingNr - 1);
+		}
+	}
+
+	const decrementRating = () => {
+		if (!ratingDown) {
+			setRatingDown(!ratingDown);
+			setRatingNr(ratingNr - 1);
+		} else {
+			setRatingDown(!ratingDown);
+			setRatingNr(ratingNr + 1);
+		}
+	}
+
+	const removeVideo = (id) => {
+		setVideosArr(videosArr.filter(video => video.id !== id));
+	}
+
+	const videoID = videoData.url.split('v=');
+
+	return (
+		<Box className='video-card' w={{ lg: '30vw', md: '45vw', sm: '70vw' }} m='3' boxShadow="lg" rounded="lg" bg='teal' overflow='hidden'>
```

Comment:
That's really good you are trying different UI tools.  I am planning to have a look tailwind next time.